src/components/experimental/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { ReactElement, ReactNode } from 'react';
+import { Heading } from 'react-aria-components';
+import styled from 'styled-components';
+import { Text, textStyles } from '../Text/Text';
+import { Modal } from '../Modal/Modal';
+import { Backdrop, BackdropProps } from '../Backdrop/Backdrop';
+import { getSemanticValue } from '../../../essentials/experimental';
+
+const Card = styled.div`
+    display: grid;
+    gap: 0.5rem;
+`;
+
+const ButtonsWrapper = styled.div`
+    padding-top: 1.5rem;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    gap: 1rem;
+`;
+
+const StyledModal = styled(Modal)`
+    width: 30rem;
```

Comment:
shouldn't this be controllable?

Reply:
In the dialog -- no, in a modal -- absolutely!